fedimint-core/src/transaction.rs

```diff
@@ -1,6 +1,6 @@
 use bitcoin::hashes::Hash as BitcoinHash;
 use bitcoin::XOnlyPublicKey;
-use fedimint_api::encoding::{Decodable, Encodable};
+use fedimint_api::encoding::{Decodable, Encodable, ModuleRegistry};
```

Comment:
I don't see any other change in this file, where is the registry used here?

Reply:
The code produced by `#[dervie(Decodable)]` now references `ModuleRegistry`. I wasn't sure if it should reference it via absolute path, hardcoding the upstream crate name, or not. Since `Decodable` and `Encodable` traits are note referenced via absolute path, and user is expected to import them manually, I did the same for `ModuleRegistry`. At least for now.

Reply:
I've looked a bit how serde does it, and it seems they are using a custom `#[serde(crate = ... )]` attribute https://serde.rs/container-attrs.html

I think for now, it might be good enough to just hardcode absolute name, and possibly get rid of the whole line `3` here (and in other files).

Reply:
#797